qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -461,6 +461,7 @@
         - gagent_check_log:
             only isa_serial
             gagent_check_type = log
+            gagent_serial_type = isa
```

Comment:
The `gagent_serial_type = isa` already be defined in L619 for variant `isa_serial`

Reply:
you're right. remove it already, thanks.